WalletWasabi.Fluent/ViewModels/Wallets/Home/History/HistoryViewModel.cs

```diff
@@ -137,13 +137,13 @@ private IEnumerable<HistoryItemViewModelBase> GenerateHistoryList(List<Transacti
 
 				balance += item.Amount;
 
-				if (item.IsLikelyCoinJoinOutput)
+				if (!item.IsLikelyCoinJoinOutput)
 				{
-					if (!item.IsConfirmed())
-					{
-						continue;
-					}
+					yield return new TransactionHistoryItemViewModel(i, item, _walletViewModel, balance, _updateTrigger);
+				}
 
+				if (item.IsLikelyCoinJoinOutput && item.IsConfirmed())
+				{
 					if (coinJoinGroup is null)
```

Comment:
Can the `coinJoinGroup` ever be null if we assigned `default` to it at LOC 132?

Reply:
yeah it can because the `null` is the default in its case.